lib/Test/APIcast.pm

```diff
@@ -153,6 +157,23 @@ sub Test::Base::Filter::env {
     return Test::Nginx::Util::expand_env_in_config($input);
 }
 
+sub Test::Base::Filter::fixture {
+    my $name = filter_arguments;
```

Comment:
Why is this different than in order filters? All the rest use `@_`.

Reply:
Because `@_` gives you the block and the data. 
http://search.cpan.org/~ingy/Test-Base-0.88/lib/Test/Base.pod#Rolling_Your_Own_Filters

`filter_arguments` gets what was passed to the filter as `filter_name=.....`.